### PR TITLE
Move filterable categories to decoreated MyContent

### PR DESCRIPTION
**Pull request by stevo**

### CODE REVIEW

File: lib/schoolkeep/scribble.rb [hunk 1]
```diff
@@ -158,7 +158,6 @@ def constantize(camel_cased_word)
 
       # Attributes
       method :disable_social_share?
-      method :filterable_categories
```
Comment (qbart) [inline lib/schoolkeep/scribble.rb]: Does this removal affects people with custom templates?
I mean can other people have problems if we do it in one phase?

Comment (stevo) [inline lib/schoolkeep/scribble.rb]: hmmmm... good point. Will check

Comment (stevo) [inline lib/schoolkeep/scribble.rb]: Yup, you were right! Reverted

**Comment by qbart** [comment]
good luck :four_leaf_clover:

**Pull request merged**